psydac/fem/projectors.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+from psydac.linalg.kron     import KroneckerDenseMatrix
+from psydac.core.interface  import matrix_multi_stages
```

Comment:
Implement  matrix_multi_stages in pure python and move them to `bsplines.py` and `bsplines_pyccel.py`